modules/containers-signature-verify-enable.adoc

```diff
@@ -7,122 +7,83 @@
 Enabling container signature validation requires files that link the registry URLs to the sigstore and then specifies the keys which verify the images.
 
 .Procedure
-. Create the files that link the registry URLs to the sigstore and that specifies the key to verify the image.
-
-** Create the `policy.json` file:
+. Create a Butane config file, `51-worker-rh-registry-trust.bu`, containing the necessary configuration for the worker nodes.
 +
-[source,terminal]
+[NOTE]
+====
+See "Creating machine configs with Butane" for information about Butane.
+====
++
+[source,yaml]
 ----
-$ cat > policy.json <<EOF
-{
-  "default": [
-    {
-      "type": "insecureAcceptAnything"
-    }
-  ],
-  "transports": {
-    "docker": {
-      "registry.access.redhat.com": [
-        {
-          "type": "signedBy",
-          "keyType": "GPGKeys",
-          "keyPath": "/etc/pki/rpm-gpg/RPM-GPG-KEY-redhat-release"
-        }
-      ],
-      "registry.redhat.io": [
-        {
-          "type": "signedBy",
-          "keyType": "GPGKeys",
-          "keyPath": "/etc/pki/rpm-gpg/RPM-GPG-KEY-redhat-release"
-        }
-      ]
-    },
-    "docker-daemon": {
-      "": [
+variant: openshift
+version: 4.8.0
+metadata:
+  name: 51-worker-rh-registry-trust
+  labels:
+    machineconfiguration.openshift.io/role: worker
+storage:
+  files:
+  - path: /etc/containers/registries.d/registry.access.redhat.com.yaml
+    mode: 0644
+    overwrite: true
+    contents:
+      inline: |
+        docker:
+          registry.access.redhat.com:
+             sigstore: https://access.redhat.com/webassets/docker/content/sigstore
+  - path: /etc/containers/registries.d/registry.redhat.io.yaml
+    mode: 0644
+    overwrite: true
+    contents:
+      inline: |
+        docker:
+          registry.redhat.io:
+            sigstore: https://registry.redhat.io/containers/sigstore
```

Comment:
These files and their contents are shipped by default on a devel RHCOS 4.9 system:

```$ ls -l /etc/containers/registries.d/
total 12
-rw-r--r--. 1 root root 894 Jun 18 19:59 default.yaml
-rw-r--r--. 1 root root 120 Jun 18 19:59 registry.access.redhat.com.yaml
-rw-r--r--. 1 root root  99 Jun 18 19:59 registry.redhat.io.yaml
$ cat /etc/containers/registries.d/registry.access.redhat.com.yaml
docker:
     registry.access.redhat.com:
         sigstore: https://access.redhat.com/webassets/docker/content/sigstore
$ cat /etc/containers/registries.d/registry.redhat.io.yaml        
docker:
     registry.redhat.io:
         sigstore: https://registry.redhat.io/containers/sigstore
$ rpm -qf /etc/containers/registries.d/registry.redhat.io.yaml
containers-common-1.2.2-10.module+el8.4.0+11311+9da8acfb.x86_64
```
I think even RHEL worker nodes would have these installed because of the requirement on `containers-common`

Since `overwrite: true` is used, it's perfectly safe to keep suggesting to write out these files, but does seem unnecessary.

Reply:
Ah, yeah.  I've removed those files, but added language that tries to imply that the default files are important, in case someone tries to use these docs as a template for configuring verification for other repositories.